### ISSUE TITLE
totp.verify docs are incorrect. Default window is 50 not 6.

### ISSUE THREAD

**Issue by gflarity** [guyht/notp#19]
totp.verify docs are wrong, says the default window is 6 but it relies on hotp.verify which has a default window of 50.

**Comment by devniel** [comment]
I agree with you. When I try to verify a token, only the ones that have a delta >= -50 are valid. I'm not sure if this delta has to do with the window property.

**Comment by naz** [comment]
Yeah, it was addressed in this PR - https://github.com/guyht/notp/pull/36 , but it never got merged in. Would be great to have it in main repository, as using forks kind of sucks :smiley:
